OneCommunityGlobal/HighestGoodNetworkApp
# fix google doc grayscale icon

# Description
Restores original version of google_dog_icon_gray.png

## Main changes explained:
- Restores original version of google_dog_icon_gray.png

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to dashboard→ Reports→ Weekly Summaries Report
6. Verify the grayscale googe doc icon appears

## Screenshots or videos of changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/23086054/cbcec6a1-80e1-41ac-b3d5-ba39f258178a)
